packages/utilities/fast-benchmarks/benchmarks/dt-add-remove/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Card, fastCard } from "@microsoft/fast-components";
+import { DOM, Observable } from "@microsoft/fast-element";
+import { DesignSystem } from "@microsoft/fast-foundation";
+
+DesignSystem.getOrCreate().register(fastCard());
+
+export default new Benchmark(
+    "queue-update",
+    function (deferred: any): void {
```

Comment:
This benchmark of adding a creating a card, adding it to the DOM, and then removing it from the DOM is seeing ~68% ops/s improvement:

```
hz change: 835.1125631303578
hz percentage change: 68.03439047252512
mean change: -0.00032984796150013643
mean percentage change: -40.48837281535488
```